src/views/Dashboard/ProgramDetails/Charts/index.tsx

```diff
@@ -96,7 +101,21 @@ function Charts(props: Props) {
     );
 
     const [showModal, setModalVisibility] = useState(false);
+    const [editableChartId, setEditableChartId] = useState<string>();
+    const [hoveredChartId, setHoveredChartId] = useState<string>();
+    const onHoverChart = useCallback(
+        (id: string) => {
+            setHoveredChartId(id);
+        },
+        [setHoveredChartId],
+    );
```

Comment:
Why is the dependency no used on the function?
Why not just do `const onHoverChart = setHoveredChartId`